docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature:`1022` Include ``error_msg`` in all api responses and place result under a corresponding ``result`` key.
```

Comment:
reword
> * :feature:`1022` Return ``errors`` field in failed API responses and place result in ``result`` field.

Reply:
Sorry this is outdated. This is no longer done. There is no `result` field.

Reply:
the changelog entry is still outdated...please adjust it according to your PR :)

Reply:
yep, will do.